### ISSUE TITLE
Analytics goal stopped recording / Event triggers not working

### ISSUE THREAD

**Issue by fickomat** [MewsSystems/public#24]
We have an issue with Analytics Goals tracking via Events as well.

It seems that the ^distributor event set as trigger for UA Event tracking triggers the tag only once (when the distributor is opened) and does not trigger at any of the next steps (as you can see bellow)
![image](https://cloud.githubusercontent.com/assets/15713916/26495253/32458528-4223-11e7-8556-7a7117123cdc.png)

I have also set up a separate tag distrubutorStepRooms that is trigered like this:
![image](https://cloud.githubusercontent.com/assets/15713916/26495380/af127ae8-4223-11e7-8602-680080a43732.png)

but it never gets fired.

**Comment by fickomat** [comment]
Can someone please take a look at this? @jirihelmich @pbambusek

**Comment by pbambusek** [comment]
Duplicate of https://github.com/MewsSystems/public/issues/23

Already fixed and waiting for deploy.
